script.js

```diff
@@ -72,4 +79,9 @@ document
     state.todos = state.todos.filter(({ checked }) => !checked);
 
     renderTodos();
+    updateSessionStorage();
   });
+
+function updateSessionStorage() {
+  sessionStorage.setItem(TODOS_SESSION_KEY, JSON.stringify(state.todos));
```

Comment:
NIT
wrapping just `sessionStorage.setItem` call into a function is a bit redundant IMHO. It might provide a little better readability of the code, but the call itself is readable quite sufficiently.